src/components/SegmentedProgressBar/SegmentedProgressBar.jsx

```diff
@@ -36,7 +38,7 @@ export default function SegmentedProgressBar({
       aria-valuemin="0"
       aria-valuemax={total}
       tabIndex="0"
-      aria-label={`Step ${current} of ${total}`}
+      aria-label={ariaLabel || `Step ${current} of ${total}`}
```

Comment:
And then this could just use the prop:

```suggestion
      aria-label={ariaLabel}
```